core/templates/pages/exploration-editor-page/exploration-save-and-publish-buttons/exploration-save-and-publish-buttons.component.spec.ts

```diff
@@ -269,4 +269,21 @@ describe('Exploration save and publish buttons component', function() {
     spyOn(editabilityService, 'isTranslatable').and.returnValue(false);
     expect($scope.isEditableOutsideTutorialMode()).toBe(false);
   });
+
+  it('should display publish button', function() {
```

Comment:
This test is almost a duplicate of the test in line 159. Shall I combine the tests?

Reply:
Yeah, please combine them

Reply:
DONE